src/helpers.php

```diff
@@ -99,9 +99,15 @@ function is_multidimensional(array $array): bool
  */
 function mkdir_recursively($path)
 {
-    $explodedPath = explode('/', $path);
+    $currentPath = getcwd();
+
+    if (DIRECTORY_SEPARATOR === '\\') {
+        $currentPath = str_replace(DIRECTORY_SEPARATOR, '/', getcwd());
```

Comment:
could we use variable instead?

```suggestion
        $currentPath = str_replace(DIRECTORY_SEPARATOR, '/', $currentPath);
```

Reply:
Sorry, forgot to change this